nvme_gw_config.py

```diff
@@ -33,3 +33,15 @@ def getint(self, section, param):
 
     def getfloat(self, section, param):
         return self.nvme_gw_config.getfloat(section, param)
+
+    def get_with_default(self, section, param, value):
+        return self.nvme_gw_config.get(section, param, fallback=value)
+
+    def getboolean_with_default(self, section, param, value):
+        return self.nvme_gw_config.getboolean(section, param, fallback=value)
+
+    def getint_with_default(self, section, param, value):
+        return self.nvme_gw_config.getint(section, param, fallback=value)
+
+    def getfloat_with_default(self, section, param, value):
+        return self.nvme_gw_config.getfloat(section, param, fallback=value)
```

Comment:
This makes me think that we should consider switching the format of the config file to YAML.  It would spare a bunch of code like this and also allow to naturally embed structured data such as `transport_{trtype}_options`.

Reply:
I am fine with switching to YAML but not in this PR please.

Reply:
Sure, if others agree with switching to YAML in the near future, `transport_tcp_options = {"max_queue_depth" : 16, "max_io_size" : 4194304, "io_unit_size" : 1048576, "zcopy" : false}` is perfectly fine as an interim solution.